src/nodes.coffee

```diff
@@ -3895,7 +3922,7 @@ exports.Code = class Code extends Base
   astProperties: (o) ->
     return Object.assign
       params: @paramForAst(param).ast(o) for param in @params
-      body: @body.ast o, LEVEL_TOP
+      body: @body.ast (merge o, checkForDirectives: yes), LEVEL_TOP
```

Comment:
Can we use `Object.assign`? I’m trying to phase out `merge` and other unnecessary helpers.

Reply:
Sure, updated to use `Object.assign()`